docs/contributing/docs-contributions/docs-components.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Docs & Blog Components
+title: Docs Components
 tableOfContentsDepth: 2
 ---
 
```

Comment:
GitHub won't let me leave comments on unchanged hidden lines but these lines below should remove mention of the blog too:

> The Gatsby docs site has a handful of components that have been developed to facilitate writing new content for ~~the blog and the~~ docs

>  A variety of components have been written to help with formatting code and content across ~~the blog and~~ the docs and don't need to be imported.